pkg/epp/controller/inferencepool_reconciler_test.go

```diff
@@ -282,7 +283,8 @@ func TestXInferencePoolReconciler(t *testing.T) {
 	if err := fakeClient.Get(ctx, req.NamespacedName, newPool1); err != nil {
 		t.Errorf("Unexpected pool get error: %v", err)
 	}
-	newPool1.Spec.TargetPortNumber = 9090
+	// TODO: change it later to 9090
+	newPool1.Spec.TargetPortNumber = 0
```

Comment:
Same, will change it back once https://github.com/kubernetes-sigs/gateway-api-inference-extension/pull/1330 gets merged.